salmon_trout/train.py

```diff
@@ -225,28 +229,28 @@ def input_fn(request_body, content_type):
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
 
-    parser.add_argument("--epochs", type=int, default=1)
+    parser.add_argument("--epochs", type=int, default=10)
```

Comment:
The advantage of using 1 as default epochs is to ensure that you don't start a longish job when you just want to test this.